src/interface/desktop/preload.js

```diff
@@ -82,5 +82,5 @@ contextBridge.exposeInMainWorld('appInfoAPI', {
 
 contextBridge.exposeInMainWorld('navigateAPI', {
   navigateToSettings: () => ipcRenderer.send('navigate', 'config.html'),
```

Comment:
Should this be also updated to navigate to the desktop app's (new) settings page?

```suggestion
  navigateToSettings: () => ipcRenderer.send('navigate', 'settings.html'),
```

Reply:
yes i was going to ask that